pointerevents/pointerevent_support.js

```diff
@@ -13,9 +13,10 @@ var All_Pointer_Events = [
 // Check for conformance to PointerEvent interface
 // TA: 1.1, 1.2, 1.6, 1.7, 1.8, 1.9, 1.10, 1.11, 1.12, 1.13
 function check_PointerEvent(event) {
+    let pointerTestName = event.pointerType + ' ' + event.type;
```

Comment:
I'm not sure if the W3C has any standard policy on this. I'd personally prefer for us to use `var` just because PEP runs this test suite and supports IE 10, so there'd be less work involved for us if we didn't have to worry about `let`.